### ISSUE TITLE
Drag and drop tests stop working in 3.6.0+

### ISSUE THREAD

**Issue by pbougie** [teamcapybara/capybara#2098]
Capybara Version: 3.6.0
Driver Version: ChromeDriver 2.41.578706
Chrome Version: 69.0.3497.81

I have various drag and drop tests that work fine in version 3.5.1. They simply stop working when Capybara is upgraded to versions 3.6.0, 3.7.0, or 3.7.1.

What has changed? I can't seem to find much information about it.

**Comment by twalpole** [comment]
Prior to 3.6.0 all that was generated for drag/drop was mousedown, mousemove, mouseup, in 3.6.0+ Capybara generates HTML5 native drag and drop events for Chrome - https://github.com/teamcapybara/capybara/commit/0538cb6688c2182995ba6beeac1384a78499263f if the element has the HTML5 `draggable` attribute set to "true".  It passes the tests in the Capybara suite for HTML5 drag and drop but it's possible it's missing something - what does your drag and drop use/require and what does the element you're dragging look like?

**Comment by pbougie** [comment]
I'm dragging `<a>` elements into `<td>` elements using the Dragula JS library. I am not using the HTML5 `draggable` attribute.

Also just tried Firefox Nightly with the same results.

**Comment by twalpole** [comment]
Below is a small ruby snippet that shows drag/drop working on the dragula example site, so if we can figure out what the difference between your usage and the dragula example is it would help narrow this down

```
require "capybara/dsl"
require 'selenium/webdriver'

sess = Capybara::Session.new(:selenium_chrome)
sess.visit('https://bevacqua.github.io/dragula/')
src = sess.first('#left-defaults > div')
dest = sess.first('#right-defaults > div')
src.drag_to dest
sleep 15 # keep session open so we can see it has dragged

```

**Comment by pbougie** [comment]
Unfortunately, it's not publicly accessible. I'll try to put something together to reproduce the problem. Thanks.

**Comment by twalpole** [comment]
After thinking about this for a while - I know why it's happening.  Images and links (the <a> elements you're dragging) default to having the HTML5 `draggable="true" `.  A workaround would be to add a `draggable="false"` attribute to all the elements you're dragging using dragula until we can fix this.  I guess I need to figure out how to tell whether another library is handling dragging -- hmmmm.

**Comment by twalpole** [comment]
The `draggable="false"` workaround won't work due to a bug in Selenium treating `draggable` it as a boolean attribute rather than the enumerated attribute that it is

**Comment by pbougie** [comment]
> The `draggable="false"` workaround won't work due to a bug in Selenium treating `draggable` it as a boolean attribute rather than the enumerated attribute that it is - [SeleniumHQ/selenium#6396](https://github.com/SeleniumHQ/selenium/issues/6396)

I was just trying it before you sent this and it wasn't working.

**Comment by twalpole** [comment]
@pbougie Please try the `drag_link` branch and see if that works correctly for you.  It should revert to the old dragging behavior if the initial mousedown event is defaultPrevented which `dragula` should be doing to disable default HTML5 drag and drop.

**Comment by pbougie** [comment]
Branch `drag_link` is working. Thanks.

**Comment by twalpole** [comment]
`drag_link` branch should be fixed for FF as well now -- Assuming tests pass I'll merge it into `master` and `3.7_stable` branches and then do a 3.7.2 release tomorrow or Thursday.

**Comment by twalpole** [comment]
Closing via 195930d900d69958647a37ea9a45634fe8d2b6bb

**Comment by pbougie** [comment]
Using `master` now. Thanks for the help.

**Comment by twalpole** [comment]
Released in 3.7.2 if you want to run master
